### DISCUSSION TITLE
Rename the Messages Window

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4045] [Ideas]
In a different post I made it was brought up that the "Messages Window" probably would benefit from being renamed, because it is not a "Window" and it also contains many tabs within it that are not all specifically all pertaining to "messages" only.
Status 
Compiler 
Message
Scribble
Terminal

If any developers are interested, maybe can name it something like "Bottom Bar" (like how Geany uses "Side Bar) or "Bottom Panel" instead, to elimate some confusion.

Thank You

**Comment by eht16** [comment]
@advice2020 would you mind linking the other discussion where it came up?

**Comment by advice2020** [comment]
Sure, it was mentioned in the following post if anyone is interested #4026

**Comment by advice2020** [comment]
Created issues posts about this that can be found here if anyone is interested.
https://github.com/geany/geany/issues/4432
